manifests/j/JanDeDobbeleer/OhMyPosh/3.176.0/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JanDeDobbeleer.OhMyPosh
+PackageVersion: 3.176.0
+PackageLocale: en-US
+Publisher: Jan De Dobbeleer
+PackageName: Oh My Posh
+License: GPL
+ShortDescription: Prompt theme engine for any shell
+Tags:
+- Console
+- Command-Line
+- Shell
+- Command-Prompt
+- PowerShell
+- WSL
+- Developer-Tools
+- Utilities
+- cli
+- cmd
+- ps
+- terminal

```

```suggestion
- Console
- Command-Line
- Shell
- Command-Prompt
- PowerShell
- WSL
- Developer-Tools
- Utilities
- cli
- cmd
- ps
- terminal
```
would you be able to make all of these lower-case in your next release?